medical_pharmacy/models/medical_pharmacist.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, tools
+from odoo.modules import get_module_resource
+
+
+class MedicalPharmacist(models.Model):
+    _name = 'medical.pharmacist'
+    _description = 'Medical Pharmacist'
+    _inherit = 'medical.abstract.entity'
+
+    @api.model
+    def _create_vals(self, vals):
+        vals.update({
+            'is_company': False,
+            'customer': False,
+        })
+        return super(MedicalPharmacist, self)._create_vals(vals)
+
+    @api.model
+    def _get_default_image(self, vals):
+        res = super(MedicalPharmacist, self)._get_default_image(vals)
```

Comment:
> Didn't we change the function _get_default_image to _allow_image_create?

Nope, that was a different method. We also moved some of the logic between the two in order to more correctly isolate the logic. https://github.com/OCA/vertical-medical/blob/10.0/medical/models/medical_abstract_entity.py#L179

> Also, I wonder if we should not have a get_default_image implemented in medical_abstract_entity then only override a get_default_image_path when inheriting.

That's pretty much just the default partner image. If nothing is returned from the inheritance chain of `_get_default_image` in the entity creation, an image isn't appended into the create values & the default partner one triggers instead. Quite an interesting little trick of the inheritance that ended up as a feature, particularly because I couldn't think of a default medical entity image 😆